src/Text/Pandoc/Writers/Markdown/Inline.hs

```diff
@@ -116,9 +124,62 @@ attrsToMarkdown attribs = braces $ hsep [attribId, attribClasses, attribKeys]
               escAttrChar '\\' = literal "\\\\"
               escAttrChar c    = literal $ T.singleton c
 
+attrsToMarkua:: Attr -> Doc Text
+attrsToMarkua attributes
+    | null list = empty
+    | length list == 1 = braces $ head list
+    |  otherwise = braces $ hcat (head list : (map (\x -> ", " <> x)
+                                                   (tail list)))
+        where attrId = case attributes of
+                        ("",_,_) -> []
+                        (i,_,_)  -> [literal $ T.append "id: " i]
+              -- all non explicit (key,value) attributes besides id are getting
+              -- a default class key to be Markua conform
+              attrClasses = case attributes of
+                             (_,[],_) -> []
+                             (_,classes,_) -> map (escAttr . ("class: " <>))
+                                classes
+              attrKeyValues = case attributes of
+                               (_,_,[]) -> []
+                               (_,_,keyvalues) -> map (\(k,v) -> escAttr k
+                                              <> ": " <>
+                                              escAttr v)
+                                              (filter (/=(T.empty,T.empty))
+                                              (map preprocessKeyValues
+                                                   keyvalues))
+              escAttr          = mconcat . map escAttrChar . T.unpack
+              escAttrChar '"'  = literal "\""
+              escAttrChar c    = literal $ T.singleton c
```

Comment:
Why not `escAttr = T.replace "\"" "\\\""`?

Reply:
We are not entirely sure where this applies, but through the process of applying your feedback we changed the entire function slightly.

Reply:
My point was that `escAttr = T.replace "\"" "\\\""` seems to do exactly what the three line definition above does.  Or am I wrong?

Reply:
We did not manage to simplify the function like that and still get the required results